requirements/systemmodel/system-model.tex

```diff
@@ -0,0 +1,24 @@
+\subsection{System Model}
+\begin{figure}
+  \subsubsection{Schedule Task}
+  \centering
+  \includegraphics[width=\linewidth]{./systemmodel/scheduletask.png}
+  \caption{This diagram depicts how a user schedules a task, which the CS distributes to WCM 1.
+  In that case CS knows that there are enough resurces on WCM 1 to execute users task.
+  At the end the CS notifies the user that the task has ended.}
```

Comment:
The image caption is inconsistent with the image.
You wrote "when task is ready" in your image but I think you meant "when task is finished".

Reply:
i will fix it.